test/parallel/test-crypto-async-sign-verify.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const crypto = require('crypto');
+const fixtures = require('../common/fixtures');
+
+async function test(
```

Comment:
this does not look like an async function at all, can you remove async and drop Promise.all?

Reply:
✅